### PR TITLE
Added documentation and comments to the files

### PR DESCRIPTION
**Pull request by yshah360**
* Added the comments to the logical pieces of the code as to what aspect is done where
* Renamed the files to suit their purpose better (and the development names were not that appropriate)

### CODE REVIEW

File: src/com/life360/ldac/design/Life360DataProvider.java [hunk 1]
```diff
@@ -83,6 +92,7 @@ public void userlookup() {
                 circleMemberIDs.add(m.id);
             }
         } catch (Exception e) {
+            e.printStackTrace();
             throw new RuntimeException("There was an error fetching the user information. Please try again");
```
Comment (pablobaxter) [inline src/com/life360/ldac/design/Life360DataProvider.java]: If it makes sense for the logging, you can add the exception as a second parameter of the `RuntimeException`

File: src/com/life360/ldac/data/CircleUserLocationMappingData.java [hunk 2]
```diff
@@ -4,6 +4,11 @@
...
 import java.util.List;
 
+/**
+ * This class contains the mapping for User, Circle and User Location Data
+ *
+ * @author yashshah
+ */
 public class CircleUserLocationMappingData {
 
     String circleId;
```
Comment (khedayatzadeh) [inline src/com/life360/ldac/data/CircleUserLocationMappingData.java]: Just a nitpick but you might want to remove the extra spacing :trollface:

File: src/com/life360/ldac/data/LDACConstants.java [hunk 3]
```diff
@@ -1,6 +1,12 @@
 package com.life360.ldac.data;
 
-public class DACTLocationConstants {
+/**
+ * This class contains the details that needs to be captured from the different
+ * Data Sources that are available.
+ *
+ * @author yashshah
+ */
+public class LDACConstants {
```
Comment (khedayatzadeh) [inline src/com/life360/ldac/data/LDACConstants.java]: Also here is as well for the extra spacing

**Comment by yshah360** [comment]
@pablobaxter I wanted to print the exception on the console just for the reference, and hence I am not throwing the original exception

**Pull request merged**